docker-compose.yml

```diff
@@ -3,6 +3,8 @@ services:
   reth:
     restart: always
     image: "ghcr.io/paradigmxyz/reth:v0.2.0-beta.2"
+    ports:
+      - 127.0.0.1:8545:8545
```

Comment:
This is why we use `localhost` now in CI? Why was this needed?

Reply:
Not this, but `network_mode: host` in `zk` container